### PR TITLE
Use manual configuration when generate go source code.

### PR DESCRIPTION
**Pull request by CPWstatic**
The default configuration of _**thrift_import**_ is out of date. It would not work well. So I replace it with a manual configuration.

### CODE REVIEW

**Comment by CPWstatic** [comment]
Jenkins go!

File: cmake/ThriftGenerate.cmake [hunk 1]
```diff
@@ -84,7 +84,7 @@ add_custom_command(
     --gen "mstch_cpp2:include_prefix=${include_prefix},process_in_event_base,stack_arguments"
     --gen "py"
     --gen "java:hashcode"
-    --gen "go"
+    --gen "go:thrift_import=github.com/facebook/fbthrift/thrift/lib/go/thrift"
```
Comment (darionyaphet) [inline cmake/ThriftGenerate.cmake]: why add the dependency when compile and generation source code？

Comment (CPWstatic) [reply cmake/ThriftGenerate.cmake]: The generated code depends on it. It has a default value, but out of date.

**Comment by CPWstatic** [comment]
Jenkins go!

File: cmake/ThriftGenerate.cmake (hunk 1 above)
Comment (darionyaphet) [reply cmake/ThriftGenerate.cmake]: But we use glide to manage the dependency.

Comment (CPWstatic) [reply cmake/ThriftGenerate.cmake]: It is not about the dependency management. I mean the configuration of **thrift_import** is explicitly insert into the generated code. like:
```import "github.com/facebook/fbthrift/thrift/lib/go/thrift"```

Comment (dutor) [reply cmake/ThriftGenerate.cmake]: @darionyaphet The default import path is _github.com/facebook/fbthrift-go_, which seems no longer exist. You could check on the output of `/path/to/the/fbthrift/generator --help`, i.e. `/opt/nebula/third-party/fbthrift/bin/thrift1 --help`

**Pull request merged**